CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [2.11.2](https://github.com/sonata-project/SonataIntlBundle/compare/2.11.1...2.11.2) - 2021-11-20
+### Fixed
+- [[#488](https://github.com/sonata-project/SonataIntlBundle/pull/488)] Correctly default to the kernel default locale when no locale is provided. ([@VincentLanglet](https://github.com/VincentLanglet))
```

Comment:
```suggestion
- [[#488](https://github.com/sonata-project/SonataIntlBundle/pull/488)] Correctly default to the kernel default locale when no locale is provided ([@VincentLanglet](https://github.com/VincentLanglet))
```

Reply:
I guess this should be changed on the branch where it was added, not on a merge.

Reply:
Yeah, if wanted, this should be modified on 2.x